lib/kitchen/verifier/inspec.rb

```diff
@@ -344,6 +344,20 @@ def runner_options_for_dockercli(config_data)
         logger.debug "Connect to Container: #{opts['host']}"
         opts
       end
+
+      # Returns a configuration Hash that can be passed to a `Inspec::Runner`.
+      #
+      # @return [Hash] a configuration hash of string-based keys
+      # @api private
+      def runner_options_for_docker(config_data)
```

Comment:
This looks identical to `#runner_options_for_dockercli`. Could we `alias_method` instead?

Reply:
@miah Yes, they are identical as of now, and I can absolutely add this change. My only concern is the kitchen-docker_cli and kitchen-docker gems are technically separate projects, and if the docker_cli gem ever needs to update these options for some reason, it could possibly break the kitchen-docker gem. What are your thoughts?

Reply:
I'm for alias_method, I think.

Reply:
If the docker_cli gem ever needs to update those options, the code can be changed at that time.